net-vpn/networkmanager-wireguard/networkmanager-wireguard-20181128.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
The version is problematic for a snapshot of a yet to release software. You should use 0_pre20181128 instead, so any eventual release version is actually considered higher by portage.

Reply:
Done.